app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/conversationList.tsx

```diff
@@ -129,6 +130,15 @@ export const List = () => {
     return () => observer.disconnect();
   }, [fetchNextPage, hasNextPage, isFetchingNextPage]);
 
+  // Handle Cmd+A / Ctrl+A to select all conversations
+  useHotkeys("mod+a", () => {
+    setAllConversationsSelected(true);
+    clearSelectedConversations(); // Clear individual selections since we're selecting all
+  }, {
+    enableOnFormTags: false, // Don't trigger when focused on form elements
+    preventDefault: true,
+  });
```

Comment:
I think it would be nicer to add an explicit parameter to `toggleAllConversations` and reuse it here. Also we can remove the comments, the code speaks for itself

```suggestion
  useHotkeys("mod+a", () => toggleAllConversations(true), {
    enableOnFormTags: false,
    preventDefault: true,
  });
```